epi_scanner/app.py

```diff
@@ -868,8 +868,8 @@ async def client_data_table(q: Q, disease: Literal["dengue", "chik"], uf: str):
 
 
 async def client_parameters(q: Q, disease: Literal["dengue", "chik"], uf: str):
-    if disease == "chik":
-        disease = "chikungunya"
+    #if disease == "chik":
```

Comment:
If it's a bug, you can just remove instead of comment

Reply:
done!